resources/gcloud.rb

```diff
@@ -9,90 +9,284 @@
 end
 
 action :install do
-  platform_cmd = Mixlib::ShellOut.new('uname')
-  platform_cmd.run_command
-  platform_cmd.error!
-  platform = platform_cmd.stdout.strip.downcase
-
-  arch_cmd = Mixlib::ShellOut.new('uname -m')
-  arch_cmd.run_command
-  arch_cmd.error!
-  arch = arch_cmd.stdout.strip
-
-  version = new_resource.version
-  if version.empty?
+  # Check infomation of operation system
+  platform = attr_platform
```

Comment:
many other attributes, methods below have attr_ prefix, this is weird to me, not sure if that's good naming, @duc4nh.